index.html

```diff
@@ -18,8 +18,9 @@
   <header>
     <div class="container">
       <div class="row">
-        <div class="column-full">
+        <div class="column-full nav-bar">
           <h1>Code Journal</h1>
+          <a class="entries-link" onclick="viewSwap('entries')">Entries</a>
```

Comment:
instead of using `onclick`, try adding event listeners by using `document.querySelector()` (or something similar) in your `main.js` file.